cmd/ttn-lw-stack/commands/root.go

```diff
@@ -109,6 +110,9 @@ var (
 			if _, err := maxprocs.Set(); err != nil {
 				logger.WithError(err).Debug("Failed to set GOMAXPROCS")
 			}
+			if _, err := memlimit.SetGoMemLimit(0.9); err != nil {
+				logger.WithError(err).Debug("Failed to set GOMEMLIMIT")
```

Comment:
Compared to `automaxprocs`, `automemlimit` does return an error when no `cgroup` limits can be found. I believe that it is fine to log this error as a debug message.

Reply:
Yes. I would log info level here though, also for max procs.